frontend/src/components/VCSProviderBasicInfoPanel.vue

```diff
@@ -60,7 +77,7 @@
   </p>
   <BBTextField
     class="mt-2 w-full"
-    :placeholder="'https://gitlab.example.com'"
+    :placeholder="instanceUrlPlaceholder"
```

Comment:
if it's github.com, then the instance URL is probably not editable.

Reply:
Addressed in https://github.com/bytebase/bytebase/pull/998/commits/91e1834f09b0a02d698198054604498159c9dcc5.